src/app/home/home.component.scss

```diff
@@ -1,107 +1,101 @@
+@import "../_variables.scss";
 
-.greetings {
-  margin-top: 30px;
-  margin-bottom: 30px;
-}
-
-.greetings-hero-img {
-  position: center;
-  height: 80%;
-  width: 80%;
-  border-radius: 50%;
-}
-
-.mobile-greetings-hero-img {
-  display: block;
-  margin-left: auto;
-  margin-right: auto;
-  position: center;
-  height: 40%;
-  width: 40%;
-  border-radius: 50%;
-}
-
-.nomargin {
-  margin: 0;
-}
-
-.grid-container {
-  margin-top: 20px;
-  margin-bottom: 20px;
+.background {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: $black;
+  z-index: -999; // hacky way to override app background color for now
```

Comment:
Can you provide some context on why this needs negative z-index to hide the background? Could it just be `display: none` instead?

Using large z-index values can quickly spiral into a z-index war that makes stacking contexts very complicated and difficult to work with (just look at all the wonkiness on the mega menu/encounter builder sidebar because of the z-index war we inherited from Cobalt).

Reply:
This isn't actually hiding the background; by default, the app's background color is either white or gray depending on the theme the user has selected. On the front page, we override that to `$black` in `.landing-page`, but on large screens `.landing-page` doesn't cover the whole screen (screenshot attached).

The `.background` div just covers the whole screen and makes the background black; eventually we'll change the global app theme.

(Is there a better way to do this, though?)

<img width="2157" alt="Screenshot 2020-05-16 at 12 28 48" src="https://user-images.githubusercontent.com/8866981/82128569-089fa400-9771-11ea-8d41-37c537a7029e.png">

Reply:
If it's just an actual background color that you want to be the full page size, then styling on `body` would probably be the simplest way to set the background color behind everything. If a descendant (like the page wrapper) wants to have a different background color from the body (like in your screenshot), then that element can just set a background color on itself and it will overlap the body by default. In general when using z-index, I always try to see if I can restructure the page before using z-index, because there's often a way to organize the markup such that z-index is rarely, if ever, needed (descendants and later siblings naturally go on top until a new stacking context is created with a non-static position or z-index).

Reply:
Interestingly, styling on body doesn't work here, since the styles are in the `HomeComponent` style sheet, and Angular doesn't allow components to affect the style of other components higher in the hierarchy than themself (and body is in the root component).

Reply:
Why does HomeComponent want to set a style for something outside of it's scope then? Perhaps this should be a responsibility of the top-level app component instead?

Reply:
Eventually it will be, yeah; the front page is kind of in a weird state right now since it's using the new styles from the design team, but the rest of the site isn't yet (that's a much bigger change).